fatal error: gnuradio/vocoder/freedv_api.h: No such file or directory

**Describe the bug**

I get this error when trying to build 0.8.2-3:

```
./gr/gr_mod_base.h:27:10: fatal error: gnuradio/vocoder/freedv_api.h: No such file or directory
   27 | #include <gnuradio/vocoder/freedv_api.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**To Reproduce**

I compile my GNUradio with the following components:

```
-- ######################################################
-- # Gnuradio enabled components
-- ######################################################
--   * testing-support
--   * volk
--   * gnuradio-runtime
--   * gr-ctrlport
--   * gr-blocks
--   * gr-fec
--   * gr-fft
--   * gr-filter
--   * gr-analog
--   * gr-digital
--   * gr-dtv
--   * gr-atsc
--   * gr-audio
--   * * alsa
--   * * oss
--   * * jack
--   * gr-channels
--   * gr-noaa
--   * gr-pager
--   * gr-trellis
--   * gr-uhd
--   * gr-video-sdl
--   * gr-vocoder
--   * gr-fcd
--   * gr-wavelet
--   * gr-zeromq
--
-- ######################################################
-- # Gnuradio disabled components
-- ######################################################
--   * python-support
--   * doxygen
--   * sphinx
--   * gnuradio-companion
--   * gr-comedi
--   * gr-qtgui
--   * gr-utils
--   * gr-wxgui
--
-- Using install prefix: /nix/store/syg7iqv95ygbwj0x2ibdl750174sqfp1-gnuradio-3.7.14.0
-- Building for version: 3.7.14.0 / 3.7.14.0
```

**Expected behavior**

I'm able to build qradiolink just fine.

**Additional context**

I'm using NixOS and I'm trying to update our qradiolink package. I'm getting this error with  0.8.3-alpha2 as well. Haven't tried to build straight from master yet.

--- a/README.md
+++ b/README.md
@@ -87,7 +87,14 @@ $ sudo apt-get install gnuradio-dev protobuf-compiler gr-osmosdr gnuradio libvol
 - Qt >= 5.14 and Qt5 development packages (older versions of Qt5 >= 5.11 might work as well)
 - qmake
 - Pulseaudio or Alsa or Jack
-- Gnuradio >= 3.8 built with UHD, SoapySDR support and FreeDV/Codec2 support. 
+- Gnuradio >= 3.8 built with UHD, SoapySDR support and FreeDV/Codec2 support. Please verify that the following was printed before you (or your distribution's packager) built gnuradio:
+<pre>
+--   * gr-vocoder
+--   * * codec2
+--   * * freedv
+</pre>
+
+In [#67](https://github.com/qradiolink/qradiolink/issues/67) it was reported that it may be necessary to [set some cmake options](https://github.com/qradiolink/qradiolink/issues/67#issuecomment-706307297) In order for [codec2](https://github.com/drowe67/codec2) and it's freedv support to be detected properly.
 - Boost and boost-devel
 - libgnuradio-osmosdr (gr-osmosdr) built with UHD, RTL-SDR, SoapySDR, HackRF, RedPitaya or BladeRF support
 - libprotobuf, libopus, libspeexdsp, libpulse-simple, libpulse, libasound, libcodec2, libsqlite3, libjpeg, libconfig++, libvolk, libftdi, libsndfile1, qtmultimedia5-dev, libqt5multimediawidgets5, libqt5multimedia5, libqt5multimedia5 (gstreamer1.0-plugins-bad for Qt video)

